src/hooks/users/useRecentVenueUsers.ts

```diff
@@ -0,0 +1,20 @@
+import { useConnectCurrentVenueNG } from "hooks/useConnectCurrentVenueNG";
+import { useVenueId } from "hooks/useVenueId";
+
+import { useRecentLocationUsers } from "./useRecentLocationUsers";
+
+export const useRecentVenueUsers = () => {
```

Comment:
[note] This hook was split out of `src/hooks/users/index.ts` into it's own file here.